tests/config-array.test.js

```diff
@@ -301,26 +301,133 @@ describe('ConfigArray', () => {
 	});
 
 	describe('Validation', () => {
-		it('should throw an error when files is not an array', async () => {
-			configs = new ConfigArray([
+
+		function testValidationError({ only = false, title, configs, expectedError }) {
+			
+			const it_ = only ? it.only : it;
```

Comment:
Can we rename `it_` to something else? Not a fan of the dangling underscore. Maybe `localIt`?

Reply:
Done in https://github.com/humanwhocodes/config-array/pull/103/commits/34f47d5034e2ebf8f4ffbfaacaaa5db5b16b8d45.